src/index.tsx

```diff
@@ -128,9 +129,16 @@ export const GitHubEmoji: React.FC<GitHubEmojiPropTypes> = ({ name }) => {
 const RepoCard: React.FC<RepoCardPropTypes> = ({
   username,
   repository,
+  useDarkTheme,
   Loading,
 }) => {
-  const [data, loadingData] = useGitHubRepo(username, repository);
+  const [data, loadingData] = useGitHubRepo(username, repository, useDarkTheme);
+  var bgColor;
```

Comment:
This can be replaced with a ternary (i.e. `const bgColor = useDarkTheme ? "#0d1117" : "white"`) and should be placed below the hooks

Reply:
GitHub is being weird, I'm referring to this block:

```ts
var bgColor;
  if (useDarkTheme == true) {
    bgColor = "#0d1117";
  } else {
    bgColor = "white"
  }
```